### ISSUE TITLE
Hard to filter certain classes through multiple word tags

### ISSUE THREAD

**Issue by yangszuyu** [jlyst/gallery#8]
---
name: Usability issue
about: Identify an issue that compromises the user experience.
title: Hard to filter certain classes through multiple word tags
labels: usability issue
assignees: 

---

When an audience is viewing, it's hard to navigate specific course that he/she is searching for.

<img width="915" alt="screen shot 2019-02-15 at 8 58 02 am" src="https://user-images.githubusercontent.com/43279908/52861414-6671b600-3100-11e9-8f8c-6b1b1123a813.png">

Suggestions: Use dropdown list to filter school name, and the topic.

**Comment by jlyst** [comment]
I agree this needs to be improved. Since it is already a dropdown in essence, would you provide an example or concept to consider?  It could be a link to an online example, screenshot of an example, or your own sketch etc.

**Comment by yangszuyu** [comment]
For the dropdown list of view, we could design the dropdown list into a hierarchical list like examples below : 
![ad-tchtrd product v1 4-dropdown](https://user-images.githubusercontent.com/43279908/53606461-e67c2f00-3b88-11e9-9459-310776e78c4a.jpg)

So, for us, the first layer could be classes, teacher names could be the second layer of the structure.
